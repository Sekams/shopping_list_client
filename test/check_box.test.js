import React from 'react';
import CheckBox from '../src/components/check_box'

test("renders the check_box component", () => {
    const wrapper = shallow(
        <CheckBox />
    );
    expect(wrapper).toMatchSnapshot();
});