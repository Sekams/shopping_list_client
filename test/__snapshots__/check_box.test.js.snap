// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the check_box component 1`] = `
<div>
  <input
    onChange={[Function]}
    type="checkbox"
  />
</div>
`;
